app/test_main.py

```diff
@@ -1,3 +1,134 @@
 from app.main import get_human_age
+import pytest
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected",
+    [
+        (0, 0, [0, 0])
+    ]
+)
+def test_zero_age(
+        cat_age: int,
+        dog_age: int,
+        expected: list[int]
+) -> None:
+    assert get_human_age(cat_age, dog_age) == expected
+
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected",
+    [
+        (14, 14, [0, 0])
+    ]
+)
+def test_slightly_below_the_first_threshold(
+        cat_age: int,
+        dog_age: int,
+        expected: list[int]
+) -> None:
+    assert get_human_age(cat_age, dog_age) == expected
+
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected",
+    [
+        (15, 15, [1, 1])
+    ]
+)
+def test_first_human_year(
+        cat_age: int,
+        dog_age: int,
+        expected: list[int]
+) -> None:
+    assert get_human_age(cat_age, dog_age) == expected
+
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected",
+    [
+        (16, 16, [1, 1])
+    ]
+)
+def test_slightly_above_the_first_threshold(
+        cat_age: int,
+        dog_age: int,
+        expected: list[int]
+) -> None:
+    assert get_human_age(cat_age, dog_age) == expected
+
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected",
+    [
+        (23, 23, [1, 1])
+    ]
+)
+def test_slightly_below_the_second_threshold(
+        cat_age: int,
+        dog_age: int,
+        expected: list[int]
+) -> None:
+    assert get_human_age(cat_age, dog_age) == expected
+
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected",
+    [
+        (24, 24, [2, 2])
+    ]
+)
+def test_second_human_year(
+        cat_age: int,
+        dog_age: int,
+        expected: list[int]
+) -> None:
+    assert get_human_age(cat_age, dog_age) == expected
+
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age, expected",
+    [
+        (28, 28, [3, 2])
+    ]
+)
+def test_slightly_above_the_second_threshold(
+        cat_age: int,
+        dog_age: int,
+        expected: list[int]
+) -> None:
+    assert get_human_age(cat_age, dog_age) == expected
+
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age",
+    [
+        (-1, 0),
+        (0, -1),
+        (-1, -1)
+    ]
+)
+def test_negative_values(
+        cat_age: int,
+        dog_age: int
+) -> None:
+    with pytest.raises(ValueError):
+        get_human_age(cat_age, dog_age)
+
+
+@pytest.mark.parametrize(
+    "cat_age, dog_age",
+    [
+        (2.4, 5),
+        (5, 2.4),
+        ("string", 5),
+        ({2}, "string"),
+        ({3: 1}, (2))
+    ]
+)
+def test_non_integer_values(
+        cat_age: int,
+        dog_age: int
+) -> None:
+    with pytest.raises(TypeError):
+        get_human_age(cat_age, dog_age)
```

Comment:
Leave one function that is dedicated to testing errors, do not adjust the `get_human_age` code